File: c++/iniciante/EX-1010.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
	
	int codPeca[2], numPeca[2];
	double vUniPeca[2], valorTotal = 0;
	float pecas[2][3];
	
    for(int i = 0; i < 2; i++){
    	cin >> codPeca[i];
    	cin >> numPeca[i];
    	cin >> vUniPeca[i];
    	for(int j = 0; j < 3; j++){
    		switch (j){
    			case 0:
    				pecas[i][j] = codPeca[i];
    			break;
    			
    			case 1:
    				pecas[i][j] = numPeca[i];
    			break;
    			
    			case 2:
    				pecas[i][j] = vUniPeca[i];
    			break;
			}
		}
	}
	
	for(int i = 0; i < 2; i++){
		valorTotal += pecas[i][1] * pecas[i][2];
	}
    
    cout << "VALOR A PAGAR: R$ " << fixed << setprecision(2) << valorTotal << endl;
    
    system("pause");
    
    return 0;
}

File: java/iniciante/ex-1010.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int[] codPeca = new int[2];
		int[] numPecas = new int[2];
		double[] valorUnitario = new double[2];
		double[][] pecas = new double[2][3];
		double valorTotal = 0;
		
		for (int i = 0; i < pecas.length; i++) {
			codPeca[i] = sc.nextInt();
			numPecas[i] = sc.nextInt();
			valorUnitario[i] = sc.nextDouble();
			
			for (int j = 0; j < pecas[i].length; j++) {
				switch (j) {
				case 0:
					pecas[i][j] = codPeca[i];
					break;
					
				case 1:
					pecas[i][j] = numPecas[i];
					break;
					
				case 2:
					pecas[i][j] = valorUnitario[i];
					break;
				}
			}
		}
		
		for (int i = 0; i < pecas.length; i++) {
			valorTotal += pecas[i][1] * pecas[i][2];
		}
		
		System.out.printf("VALOR A PAGAR: R$ %.2f", valorTotal);
		System.out.println();
		
		sc.close();

	}

}
